src/main/scala/cr/pulsar/Consumer.scala

```diff
@@ -78,7 +78,7 @@ object Consumer {
           c.nack(id) *> (new IllegalArgumentException("Decoding error")).raiseError[F, E]
       }
 
-      logAction(data)(Topic.Name(m.getTopicName)) &> acking
+      logAction(data)(Topic.URL(m.getTopicName)) &> acking
```

Comment:
Isn't it really a topic name here?

Reply:
It is actually the URL but also contains the name. See how we create the consumer: https://github.com/cr-org/neutron/blob/master/src/main/scala/cr/pulsar/Consumer.scala#L34